pkg/cli/shim_kubectl.go

```diff
@@ -43,3 +47,19 @@ func shimKubectlForOc() {
 	versioned.GeneratorFn = originpolymorphichelpers.NewGeneratorsFn(versioned.GeneratorFn)
 	describeversioned.DescriberFn = originpolymorphichelpers.NewDescriberFn(describeversioned.DescriberFn)
 }
+
+func oAPIToGroupified(uncast runtime.Object, gvk *schema.GroupVersionKind) {
+	before := gvk.Group
+	legacygroupification.OAPIToGroupified(uncast, gvk)
+	if before != gvk.Group {
```

Comment:
Deprecated and will be removed in a future release

Reply:
:+1: updated